README.md

```diff
@@ -29,10 +37,15 @@ You will need the following things properly installed on your computer.
 
 Make use of the many generators for code, try `ember help generate` for more details
 
-### Running Tests
+### Running Tests and generating coverage
+
+To run the testuise, you have to install `karma-cli`:
+
+* `npm install -g karma-cli`
+
+Now, to run the testsuite you now can use the following command. This will also generate the coverage:
 
-* `ember test`
-* `ember test --server`
+* `karma start`
```

Comment:
Because `npm`'s `test` command is configured to run `karma start` it's not necessary to install `karma-cli` globally or to execute `karma start` to run the tests. Executing `npm test` is enough.